internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/grub.go

```diff
@@ -44,6 +56,79 @@ func NewConfig() *Config {
 	}
 }
 
+// KexecLoad does a kexec using the bootloader.
+func (c *Config) KexecLoad(r runtime.Runtime, disk string) error {
+	return mount.PartitionOp(
+		disk,
+		[]mount.Spec{
+			{
+				PartitionLabel: constants.BootPartitionLabel,
+				FilesystemType: partition.FilesystemTypeXFS,
```

Comment:
nit: this should perhaps have a constant somewhere, as that's not necessarily XFS but rather we create it as XFS

Reply:
didn't quite get that

Reply:
I think Dmitry's point is that GRUB has this multiple types around the code (as it does similar mount operations), so the whole block might be shared?